### PR TITLE
using sandbox skeleton

### PR DESCRIPTION
**Pull request by TrevorOctober**

### CODE REVIEW

**Comment by TrevorOctober** [comment]
Not ready for review

**Comment by TrevorOctober** [comment]
@rudigiesler | @smn Please review

File: docs/tutorial/sandbox_skeleton.rst [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
+Find the `sandbox skeleton <https://github.com/praekelt/go-jsbox-skeleton>`_ repo.
+
+Clone the repo::
+
+	$ git clone https://github.com/praekelt/go-jsbox-skeleton
+	$ cd go-jsbox-skeleton
+
+Install requirements::
+
+    $ npm Install
```
Comment (smn) [inline docs/tutorial/sandbox_skeleton.rst]: I think this needs to be lowercase `install`.

**Comment by smn** [comment]
other than the typo, :+1:

**Pull request merged**